### PR TITLE
Make sure the timestamp is updated.

### LINKED ISSUE #97: The timestamp of the collection is not updated when kinto-signer set the signature
When copying staging/certificates data to blocklists/certificates I realized that the blocklists/certificates collection wasn't updated.

We also need to make sure that the timestamp of staging/certificates is modified when we turn the status back to signed.

### PR DESCRIPTION
**Pull request by Natim**
Fixes #97 

r? @leplatrem

### CODE REVIEW

File: kinto_signer/updater.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def push_records_to_destination(self):
                     object_id=record['id'],
                     record=record)
 
-    def set_destination_signature(self, signature):
+    def set_destination_signature(self, signature, last_modified=None):
```
Comment (leplatrem) [inline kinto_signer/updater.py]: From reading the diff, it looks like we don't need to have a default value to `None` here. `set_destination_signature()` is always called with `last_modified` in `sign_and_update_destination()` #KISS

File: kinto_signer/updater.py [hunk 2]
```diff
@@ -183,7 +188,7 @@ def set_destination_signature(self, signature):
             object_id=self.destination['collection'],
             record=collection_record)
 
-    def update_source_status(self, status):
+    def update_source_status(self, status, last_modified=None):
```
Comment (leplatrem) [inline kinto_signer/updater.py]: From reading the diff, it looks like the `last_modified` is never used. I don't understand why we would need enforcing the source collection last_modified to a particular timestamp actually :)

**Comment by leplatrem** [comment]
The timestamp is always updated, that's something guaranteed by the storage backend.

I guess the problem is that we want to set it to a particular value, for example, the timestamp of the source collection. But this is not what this patch does, it takes the timestamp of the source list of records, not the timestamp of source collection object.

I'm not against this change, but it deserves a bit of commenting ;)

File: kinto_signer/updater.py (hunk 1 above)
Comment (Natim) [inline kinto_signer/updater.py]: Yes why not.

File: kinto_signer/updater.py (hunk 2 above)
Comment (Natim) [inline kinto_signer/updater.py]: It is used here: https://github.com/Kinto/kinto-signer/pull/98/files/9de198711522332f87126bddeeb5a766a78d1d53#diff-79ac4263d72422db0d795ceb4998ee4cR203

**Comment by Natim** [comment]
> The timestamp is always updated, that's something guaranteed by the storage backend.

Unfortunately that's not the case, if the last_modified value is given it won't change. Reason why it is broken.

We could remove the value to let the backend choose, or use it to set the collection_timestamp to the last_modified value :)

**Comment by leplatrem** [comment]
>> The timestamp is always updated, that's something guaranteed by the storage backend.
>
> Unfortunately that's not the case, if the last_modified value is given it won't change. Reason why it is broken.

It will always change, even if `last_modified` is given. If less or equal that current it will be bumped as usual, otherwise it will use the provided value. If you found a bug with that, we should fix it in Kinto first!

File: kinto_signer/updater.py (hunk 2 above)
Comment (leplatrem) [inline kinto_signer/updater.py]: What I meant is that the caller never provides it sorry

**Comment by leplatrem** [comment]
> It will always change, even if last_modified is given. If less or equal that current it will be bumped as usual, otherwise it will use the provided value. If you found a bug with that, we should fix it in Kinto first!

Ok, actually you were right. When going through the storage backend directly, the last_modified can be enforced:

* https://github.com/Kinto/kinto/blob/3.2.1/kinto/tests/core/test_storage.py#L613-L617
* the behaviour I was describing is done in the resource part: https://github.com/Kinto/kinto/blob/3.2.1/kinto/core/resource/__init__.py#L592-L595 which is bypassed here

**Comment by leplatrem** [comment]
Please update changelog, r+ otherwise

**Pull request merged**